HOSTING.md

```diff
@@ -0,0 +1,340 @@
+OZONE_SERVICE_ACCOUNT_HANDLE="mylabeler.bsky.social"
+OZONE_SERVER_DID="$(curl --fail --silent --show-error "https://api.bsky.app/xrpc/com.atproto.identity.resolveHandle?handle=${OZONE_SERVICE_ACCOUNT_HANDLE}" | jq --raw-output .did)"
+OZONE_ADMIN_PASSWORD="$(openssl rand --hex 16)"
+OZONE_SIGNING_KEY_HEX="$(openssl ecparam --name secp256k1 --genkey --noout --outform DER | tail --bytes=+8 | head --bytes=32 | xxd --plain --cols 32)"
+POSTGRES_PASSWORD="..." # Use password from postgres env setup
+
+cat <<OZONE_CONFIG | sudo tee /ozone/ozone.env
+OZONE_SERVER_DID=${OZONE_SERVER_DID}
+OZONE_PUBLIC_URL=https://${OZONE_HOSTNAME}
+OZONE_ADMIN_DIDS=${OZONE_SERVER_DID}
+OZONE_ADMIN_PASSWORD=${OZONE_ADMIN_PASSWORD}
+OZONE_SIGNING_KEY_HEX=${OZONE_SIGNING_KEY_HEX}
+OZONE_DB_POSTGRES_URL=postgresql://postgres:${POSTGRES_PASSWORD}@localhost:5432/ozone
+OZONE_DB_MIGRATE=1
+OZONE_DID_PLC_URL=https://plc.directory
+OZONE_APPVIEW_URL=https://api.bsky.app
+OZONE_APPVIEW_DID=did:web:api.bsky.app
+LOG_ENABLED=1
+OZONE_CONFIG
+```
+
+#### Start the Ozone containers
+
+##### Download the Docker compose file
+
+Download the `compose.yaml` to run your Ozone instance, which includes the following containers:
+
+- `ozone` Node Ozone server—both UI and backend—running on http://localhost:3000
+- `postgres` Postgres database used by the Ozone backend
+- `caddy` HTTP reverse proxy handling TLS and proxying requests to Ozone
+- `watchtower` Daemon responsible for auto-updating containers to keep the server secure and current
+
+```bash
+curl https://raw.githubusercontent.com/bluesky-social/ozone/main/service/compose.yaml | sudo tee /ozone/compose.yaml
+```
+
+##### Create the systemd service
+
+```bash
+cat <<SYSTEMD_UNIT_FILE | sudo tee /etc/systemd/system/ozone.service
+[Unit]
+Description=Bluesky Ozone Service
+Documentation=https://github.com/bluesky-social/ozone
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+WorkingDirectory=/ozone
+ExecStart=/usr/bin/docker compose --file /ozone/compose.yaml up --detach
+ExecStop=/usr/bin/docker compose --file /ozone/compose.yaml down
+
+[Install]
+WantedBy=default.target
+SYSTEMD_UNIT_FILE
+```
+
+##### Start the service
+
+**Reload the systemd daemon to create the new service:**
+
+```bash
+sudo systemctl daemon-reload
+```
+
+**Enable the systemd service:**
+
+```bash
+sudo systemctl enable ozone
+```
+
+**Start the ozone systemd service:**
+
+```bash
+sudo systemctl start ozone
+```
+
+**Ensure that containers are running**
+
+There should be a caddy, ozone, postgres, and watchtower container running.
+
+```bash
+sudo systemctl status ozone
+```
+
+```bash
+sudo docker ps
+```
+
+### Verify that Ozone is online
+
+You can check if your server is online and healthy by requesting the healthcheck endpoint, and by visiting the UI in browser at https://ozone.example.com/.
+
+```bash
+curl https://ozone.example.com/xrpc/_health
+{"version":"0.2.2-beta.2"}
```

Comment:
the version for mine is `0.0.17-next.1`. probably worth updating it just so it isn't confusing